packages/gatsby-plugin-google-analytics/src/gatsby-ssr.js

```diff
@@ -78,7 +78,7 @@ export const onRenderBody = (
   if(${
     typeof pluginOptions.respectDNT !== `undefined` &&
     pluginOptions.respectDNT == true
-      ? `!(navigator.doNotTrack == "1" || window.doNotTrack == "1")`
+      ? `!(parseInt(navigator.doNotTrack) === 1 || parseInt(window.doNotTrack) === 1 || parseInt(navigator.msDoNotTrack) === 1 || navigator.doNotTrack === "yes")`
```

Comment:
We can move to `==` instead of `===` which fixes your issue too with fewer bytes.

```suggestion
      ? `!(navigator.doNotTrack == '1' || window.doNotTrack) == '1' || navigator.msDoNotTrack == '1' || navigator.doNotTrack === "yes")`
```

Reply:
Thanks for the review. The reason of `parseInt()` is the possible [extension of doNotTrack](https://www.w3.org/TR/tracking-dnt/#dnt-extensions) value, which could be e.g. `1xyz`, so this way the value is parsed correctly. What do you think?

Reply:
Oh make sense! 👍 thanks for clarifying.